WordPress/gutenberg
# Image: Update image block margins on RTL

## What?

Adds two additional `/*rtl:ignore*/` to adjust the margin of the image block on RTL.

Closes https://github.com/WordPress/gutenberg/issues/44845
Credit to @sabernhardt

## Why?

In RTL languages, image blocks that are aligned to the left or right can touch the text, because the margin that is intended to prevent this, is on the incorrect side of the image.


## Testing Instructions

1. Activate Twenty Sixteen.
2. Open the post editor.
3. Add an image block with a small image.
4. Set the image block's alignment to Align left.
5. Add a paragraph block immediately following the image.
6. Create another image block, aligned right, and follow that with another paragraph.
8. Save the post
9. Switch to an RTL language.
10. View the result on the front end. The images should not touch the text.

## Screenshots or screencast 
After, left aligned image has right margin:
<img width="944" alt="Screenshot 2023-01-31 at 14 55 03" src="https://user-images.githubusercontent.com/7422055/215793071-f4f6e8e0-18c3-4154-80fc-35184aba4f5a.png">

After, right aligned image has left margin:
<img width="944" alt="Screenshot 2023-01-31 at 14 55 23" src="https://user-images.githubusercontent.com/7422055/215792885-0b79831d-5e06-4e47-8335-e064cf96638a.png">
